lib/bugcrowd_templates.rb

```diff
@@ -40,15 +40,52 @@ def get(
       file_name: file_name
     ).template_file(type)
 
-    template_data(template_path)
+    template_data(file_path: template_path, category: category, subcategory: subcategory, item: item)
   end
   # rubocop:enable Metrics/ParameterLists
 
   def current_directory
     DATA_DIR
   end
 
-  def template_data(file_path)
-    File.open(file_path).read if File.exist?(file_path)
+  def template_data(file_path:, category:, subcategory:, item:)
+    if File.exist?(file_path)
+      File.open(file_path).read
+    else
+      find_template(file_path, category, subcategory, item)
+    end
+  end
+
+  def find_template(file_path, category, subcategory, item)
```

Comment:
Why do we need to jump through this function just to get to presented_template? This seems redundant?

Reply:
i have removed those methods